ethereum/wait_for_event_task_test.go

```diff
@@ -91,48 +82,48 @@ func checkKwargs(t *testing.T, task *WaitForEventTask, kwargs map[string]interfa
 	assert.Error(t, err)
 }
 
-func TestWaitForEventTask_RunTask(t *testing.T) {
-	task := new(WaitForEventTask)
-	jm := testingjobs.MockJobManager{}
-	jm.On("UpdateTaskStatus", mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(nil)
-	task.BaseTask = jobsv1.BaseTask{
-		JobManager: jm,
-	}
-	task.ethContextInitializer = func() (ctx context.Context, cancelFunc context.CancelFunc) {
-		t := time.Now().Add(5 * time.Second)
-		return context.WithDeadline(context.Background(), t)
-	}
-
-	// deadline exceeded
-	task.filterLogsFunc = func(ctx context.Context, query ethereum.FilterQuery) (logs []types.Log, err error) {
-		return nil, context.DeadlineExceeded
-	}
-	_, err := task.RunTask()
-	assert.Equal(t, err, gocelery.ErrTaskRetryable)
-
-	// non retryable
-	task.filterLogsFunc = func(ctx context.Context, query ethereum.FilterQuery) (logs []types.Log, err error) {
-		return nil, errors.New("random error")
-	}
-	_, err = task.RunTask()
-	assert.Error(t, err)
-	assert.Equal(t, err.Error(), "[random error]")
-
-	// no logs
-	task.filterLogsFunc = func(ctx context.Context, query ethereum.FilterQuery) (logs []types.Log, err error) {
-		return nil, nil
-	}
-	_, err = task.RunTask()
-	assert.Error(t, err)
-	assert.Equal(t, err, gocelery.ErrTaskRetryable)
-
-	// success
-	task.filterLogsFunc = func(ctx context.Context, query ethereum.FilterQuery) (logs []types.Log, err error) {
-		return []types.Log{
-			{},
-		}, nil
-	}
-	_, err = task.RunTask()
-	assert.NoError(t, err)
-	jm.AssertExpectations(t)
-}
+// func TestWaitForEventTask_RunTask(t *testing.T) {
```

Comment:
Is this unit test not relevant anymore?

Reply:
this old v1 celery task will be removed soon